src/main/scala/org/ergoplatform/nodeView/wallet/scanning/Scan.scala

```diff
@@ -19,11 +19,14 @@ import scala.util.{Failure, Success, Try}
   * @param scanName       - scan description (255 bytes in UTF-8 encoding max)
   * @param trackingRule  - a predicate to scan the blockchain for specific scan-related boxes
   * @param walletInteraction - a flag which is prescribing how the scan is interacting with the p2pk-wallet
+  * @param removeOffchain - a flag which prescribing whether box spent offchain should be removed from unspent boxes,
+  *                        see https://github.com/ergoplatform/ergo/issues/1314 , true by default
   */
 case class Scan(scanId: ScanId,
                 scanName: String,
                 trackingRule: ScanningPredicate,
-                walletInteraction: ScanWalletInteraction.Value)
+                walletInteraction: ScanWalletInteraction.Value,
+                removeOffchain: Boolean)
```

Comment:
```suggestion
                removeOffchain: Boolean = true)
```
to be in sync with the doc comment.